webapp/src/widgets/Page/Page.js

```diff
@@ -1,11 +1,7 @@
 import React from 'react'
-import Container from 'react-bootstrap/Container'
 import './page.css'
+import { Container } from 'react-bootstrap'
```

Comment:
the above removed change was good.
We should follow this approach.
Instead of importing entire component, just import the required one only.

Reply:
Udpated.